### PR TITLE
change the token separator space to dot

### PR DESCRIPTION
**Pull request by XJangel**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

/kind bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #
command line parsing error when using parameter token

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: cloud/pkg/cloudhub/servers/httpserver/server.go [hunk 1]
```diff
@@ -88,7 +88,7 @@ func edgeCoreClientCert(w http.ResponseWriter, r *http.Request) {
 		w.Write([]byte(fmt.Sprintf("Invalid authorization token")))
 		return
 	}
-	bearerToken := strings.Split(authorizationHeader, " ")
+	bearerToken := strings.Split(authorizationHeader, ".")
 	if len(bearerToken) != 2 {
```
Comment (fisherxu) [inline cloud/pkg/cloudhub/servers/httpserver/server.go:92]: `if len(bearerToken) != 4 ` ?

Line 97: bearerToken[1] -> bearerToken[1:]?

File: edge/pkg/edgehub/process.go [hunk 2]
```diff
@@ -43,7 +43,7 @@ func (eh *EdgeHub) applyCerts() error {
 	}
 
 	// validate the CA certificate by hashcode
-	tokenParts := strings.Split(config.Config.Token, " ")
+	tokenParts := strings.Split(config.Config.Token, ".")
 	if len(tokenParts) != 2 {
```
Comment (fisherxu) [inline edge/pkg/edgehub/process.go:47]: !=4?

line 69: tokenParts[1:] ?

File: cloud/pkg/cloudhub/servers/httpserver/server.go [hunk 3]
```diff
@@ -88,13 +88,13 @@ func edgeCoreClientCert(w http.ResponseWriter, r *http.Request) {
 		w.Write([]byte(fmt.Sprintf("Invalid authorization token")))
 		return
 	}
-	bearerToken := strings.Split(authorizationHeader, " ")
-	if len(bearerToken) != 2 {
+	bearerToken := strings.Split(authorizationHeader, ".")
+	if len(bearerToken) != 4 {
```
Comment (fisherxu) [inline cloud/pkg/cloudhub/servers/httpserver/server.go:92]: We need revert here ;-)

File: cloud/pkg/cloudhub/servers/httpserver/server.go [hunk 4]
```diff
@@ -88,13 +88,13 @@ func edgeCoreClientCert(w http.ResponseWriter, r *http.Request) {
...
 		w.WriteHeader(http.StatusUnauthorized)
 		w.Write([]byte(fmt.Sprintf("Invalid authorization token")))
 		return
 	}
-	token, err := jwt.Parse(bearerToken[1], func(token *jwt.Token) (interface{}, error) {
+	token, err := jwt.Parse(strings.Join(bearerToken[1:], "."), func(token *jwt.Token) (interface{}, error) {
```
Comment (fisherxu) [inline cloud/pkg/cloudhub/servers/httpserver/server.go:97]: ditto

File: cloud/pkg/cloudhub/servers/httpserver/server.go (hunk 3 above)
Comment (XJangel) [reply cloud/pkg/cloudhub/servers/httpserver/server.go]: got it!

**Pull request merged**